src/main/java/org/team1540/robot2019/commands/auto/TurnUntilNewTarget.java

```diff
@@ -23,7 +23,9 @@
 
     private static final Logger logger = Logger.getLogger(TurnUntilNewTarget.class);
     private static final double MAX_ANGULAR_VEL = 5;
-    private static double ANGULAR_VEL = 2.5;
+    private static double FAST_ANGULAR_VEL = 7;
```

Comment:
In a perfect world we'd have a centralized location for all the tuning but I really do not wanna deal with that so let's just jot that down as something to do better next year